config.ru

```diff
@@ -1,10 +1,13 @@
 require './app'
 
+set :database, ENV['DATABASE_URL'] || 'postgres://localhost/[YOUR_DATABASE_NAME]'
```

Comment:
Surething.  I believe that was added in to help support Boxen for local development.  Specifically, how Boxen sets Postgres to run on a non-standard port.